packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -88,8 +88,7 @@ export class OrgAuthInfo {
   public static async getUsername(
     usernameOrAlias: string
   ): Promise<string | undefined> {
-    const username = await Aliases.fetch(usernameOrAlias);
-    return Promise.resolve(username);
+    return (await Aliases.fetch(usernameOrAlias)) || usernameOrAlias;
```

Comment:
If a username doesn't have an alias, it now just returns `usernameOrAlias` which should be a username unless it's undefined.

Reply:
This change looks good! I think because this method was returning undefined before, we have this check being done in orgList, metadataCmp, and metadataType too. Should we get rid of the duplicated check?

Reply:
Ah good point, I can rework those a bit.